config/v1/types_feature.go

```diff
@@ -131,6 +131,7 @@ var FeatureSets = map[FeatureSet]*FeatureGateEnabledDisabled{
 			"TopologyManager",                // sig-pod, sjenning
 			"NodeDisruptionExclusion",        // sig-scheduling, ccoleman
 			"ServiceNodeExclusion",           // sig-scheduling, ccoleman
+			"SCTPSupport",
```

Comment:
you need to provide a sig (or openshift team) and the name of the maintainer we will nag if it breaks or if we decide to remove it.

Reply:
@karmab I think we can take inspiration from what was done with the default policy